test/includes/microcloud.sh

```diff
@@ -1279,17 +1279,24 @@ setup_system() {
     # Remove unneeded/unwanted packages
     lxc exec "${name}" -- apt-get autopurge -y lxd-installer
 
+    packages="jq"
+
     # Install the snaps.
     # Retry the attempts in case the download from external sources fails due to instability.
     retry lxc exec "${name}" -- apt-get update
     if [ -n "${CLOUD_INSPECT:-}" ] || [ "${SNAPSHOT_RESTORE}" = 0 ]; then
-      retry lxc exec "${name}" -- apt-get install --no-install-recommends -y jq zfsutils-linux htop
+      packages+=" zfsutils-linux htop"
```

Comment:
I've never used `htop`, IIRC it was Max's preference over the regular `top`. I wouldn't mind dropping it at one point.

Reply:
Yep